CONTRIBUTING.md

```diff
@@ -96,6 +96,19 @@ Then from the root directory, you can run:
 
   Note: this is not through turbo, so you'll have to build turbo yourself first.
 
+- Updating Integration Tests
+
+  ```
+  pnpm -- turbo run build --filter=cli
+  pnpm test:interactive -F turborepo-tests-integration
+  ```
+
+  You can pass a test name to run a single test, or a directory to run all tests in that directory.
+
+  ```
+  pnpm test:interactive -F turborepo-tests-integration -- run-summary.t
```

Comment:
this doesn't work because the `test:interactive` command already has a `tests` argument

```
    "test:interactive": "PRYSK_INTERACTIVE=true prysk tests",
```

```console
~/src/github.com/vercel/turbo$ pnpm --filter turborepo-tests-integration test:interactive -- run-summary.t

> turborepo-tests-integration@ test:interactive ~/src/github.com/vercel/turbo/turborepo-tests/integration
> PRYSK_INTERACTIVE=true prysk tests "--" "run-summary.t"

Running ~/src/github.com/vercel/turbo/turborepo-tests/integration/.cram_env/bin/prysk --shell=bash --interactive  --
usage: prysk [OPTIONS] TESTS...
prysk: error: the following arguments are required: TESTS
~/src/github.com/vercel/turbo/turborepo-tests/integration:
 ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  turborepo-tests-integration@ test:interactive: `PRYSK_INTERACTIVE=true prysk tests "--" "run-summary.t"`
Exit status 1
```

Reply:
Weird. I use this all the time, but I guess it works only in bun?

Reply:
Yep. Bun only because `pnpm` passes the `--` directly

Reply:
sounds great!